src/seismometer/data/pandas_helpers.py

```diff
@@ -267,6 +464,17 @@ def event_name(event: str) -> str:
     return event
 
 
+def event_value_name(event_value: str) -> str:
+    """Converts event value count column into the event value name."""
+    val = event_value
+    if "~" in val:
+        val = val.split("~")[1]
+    if val.endswith("_Count"):
+        val = val[:-6]
+
+    return val
```

Comment:
We have a bunch of these string converters, can we centralize them into one method that explains the different columns associated with an event and make them easier to understand/use/test?

Reply:
I agree with you that the core methods region is starting to get crowded and consolidating the string converters would be nice.

Does that make sense to do in this PR? Feels like it would have a decent amount of design lift and we'd still need the current functions to exist as wrappers as to not break existing seismographs.

Reply:
wrote up as an issue